backend/routes/users.js

```diff
@@ -18,4 +18,17 @@ router.get("/", [authorizeUser(["roster"])], (req, res, next) => {
     });
 });
 
+/**
+ * Edit user information
+ */
+router.post("/", [authorizeUser([])], (req, res, next) => {
```

Comment:
Should this be a put request since we are editing user information?

Reply:
I agree! Thanks for catching that. I'll make that change now.